src/main/kotlin/commands/BanCommand.kt

```diff
@@ -166,21 +166,32 @@ object BanCommand : Command("ban") {
             return
         }
 
-        user.getPrivateChannel().send {
-            embed {
-                field(
-                    "You were banned by:",
-                    message.author!!.mention,
-                    false
-                )
-                field(
-                    "Ban Reason:",
-                    fixedReason,
-                    false
-                )
-                color = Colors.error
+        try {
+            user.getPrivateChannel().send {
+                embed {
+                    field(
+                        "You were banned by:",
+                        message.author!!.mention,
+                        false
+                    )
+                    field(
+                        "Ban Reason:",
+                        fixedReason,
+                        false
+                    )
+                    color = Colors.error
+                }
+            }
+        } catch(e: Exception){
+            message.channel.send {
+                embed {
+                    title = "Error"
+                    description = "I couldn't DM that user the ban reason, they might have had DMs disabled."
+                    timestamp
```

Comment:
Remove the timestamp, it makes it more inconsistent from other embeds the bot sends